scripts/requirements-win10.txt

```diff
@@ -0,0 +1,8 @@
+python=3.8.2
```

Comment:
Rename the file to `tool-versions-win10.txt`

Reply:
done